index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+
+<body>
+
+  <header>
+    <nav>
+      <a href="#"><img src="images/MyLogo.png" alt="Logo"></a>
+      <a href="#"> <img src="images/hamb.png" alt="hamb" class="humb"> </a>
```

Comment:
- [ ] The hamburger on the right should carry the same design i.e colour as the template design.
_See the images below for clarity._

**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/121541250-fd429a80-c9bb-11eb-96b4-3f5c091f94a5.png) | ![image](https://user-images.githubusercontent.com/49621279/121541325-0cc1e380-c9bc-11eb-9190-7d24aa996b8b.png)